packages/gatsby/src/internal-plugins/dev-404-page/raw_dev-404-page.js

```diff
@@ -20,7 +20,7 @@ class Dev404Page extends React.Component {
 
     this.state = {
       hasMounted: false,
-      showCustom404: false,
+      showCustom404: process.env.GATSBY_DISABLE_CUSTOM_404 || false,
```

Comment:
I think GATSBY_DISABLE_CUSTOM_404 will be `"true"` and not `true`. It will still work but just wanted to point that out if you want to convert it to a boolean

Reply:
Jarmo added this helper to handle `true`/`false` strings and usual potential values for "boolean" env vars - https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-core-utils/src/is-truthy.ts